src/model.ts

```diff
@@ -1,5 +1,5 @@
 /*!
- * Copyright 2019 Google Inc. All Rights Reserved.
+ * Copyright 2020 Google Inc. All Rights Reserved.
```

Comment:
We're supposed to keep these dates to match the original introduction of the file, not the latest year.  We should keep this as is!